cirq_superstaq/ops/qubit_gates_test.py

```diff
@@ -108,6 +108,69 @@ def test_zz_swap_parameterized() -> None:
         _ = cirq.pauli_expansion(gate)
 
 
+def test_stripped_cz_gate() -> None:
+    phi = 0.123
+    gate = css.StrippedCZGate(phi)
+    assert cirq.has_unitary(gate)
+
+    assert str(gate) == "StrippedCZGate(0.123)"
+    assert repr(gate) == "css.StrippedCZGate(0.123)"
+    cirq.testing.assert_equivalent_repr(gate, setup_code="import cirq_superstaq as css")
+    expected = np.diag(([1.0, np.exp(1j * phi), np.exp(1j * phi), np.exp(1j * (2 * phi - np.pi))]))
+    assert np.allclose(cirq.unitary(gate), expected)
+
+    qubits = cirq.LineQubit.range(3)
+    operation = gate(qubits[0], qubits[2])
+    assert cirq.decompose_once(operation) == [
+        cirq.rz(phi).on(qubits[0]),
+        cirq.rz(phi).on(qubits[2]),
+        cirq.CZ(qubits[0], qubits[2]),
+    ]
+    cirq.testing.assert_has_consistent_apply_unitary(gate)
+    cirq.testing.assert_decompose_is_consistent_with_unitary(gate, ignoring_global_phase=True)
+    cirq.testing.assert_consistent_resolve_parameters(gate)
+
+    assert cirq.approx_eq(css.StrippedCZGate(1.23), css.StrippedCZGate(1.23 + 2 * np.pi))
+    assert cirq.approx_eq(css.StrippedCZGate(np.pi + 1e-10), css.StrippedCZGate(np.pi - 1e-10))
+
+    assert gate**1 == gate
```

Comment:
```suggestion
    assert gate**1 == gate
    assert gate**-1 == css.StrippedCZGate(-gate.phi)
```
(assuming prior suggestion)